security/acme-client/src/opnsense/mvc/app/controllers/OPNsense/AcmeClient/forms/settings.xml

```diff
@@ -1,47 +0,0 @@
-<form>
```

Comment:
oops, I think you should not remove this file :)

Reply:
Tried t remote it from the commit using the web interface - my bad. Do I just read the original back?